release.nix

```diff
@@ -30,8 +30,7 @@ let
             docbook5 docbook5_xsl
             autoconf-archive
             git
-            libseccomp
-          ];
+          ] ++ lib.optional stdenv.isLinux libseccomp;
```

Comment:
I think if we're doing this per-system, we'd need to use the `system` parameter here, not `stdenv`?